pkg/ccl/streamingccl/streamclient/random_stream_client.go

```diff
@@ -223,7 +224,7 @@ func (m *randomStreamClient) getDescriptorAndNamespaceKVForTableID(
 		IngestionDatabaseID,
 		tableID,
 		fmt.Sprintf(RandomStreamSchemaPlaceholder, tableName),
-		&descpb.PrivilegeDescriptor{},
+		descpb.NewDefaultPrivilegeDescriptor(security.RootUserName()),
```

Comment:
Now that we don't insta-upgrade our privilege descriptors when validating, we have to be more careful when crafting test cases.